### PR TITLE
Deprecate Viki::Container.recommendations and Viki::Video.recommendations

### PR DESCRIPTION
**Pull request by piratefsh**
See Asana task for context: https://app.asana.com/0/95160246791514/49859087939409

### CODE REVIEW

**Comment by piratefsh** [comment]
FYI @allenlsy @uohzxela

File: lib/viki/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module Viki
-  VERSION = "5.0.8"
+  VERSION = "5.1.0"
```
Comment (allenlsy) [inline lib/viki/version.rb]: actually we can make it 5.0.9, or even 5.0.11 is acceptable. You can re-consider the version number

File: spec/viki/video_spec.rb [hunk 2]
```diff
@@ -1,22 +1,14 @@
...
-  # end
-
-  # it "fetches recommendations for episode 1 of BoF" do
-  #   resp = stub
-  #   Viki::Video.should_receive(:fetch).with(recommended_for: '44699v').and_yield(resp)
-  #   Viki::Video.recommendations("44699v") do |response|
-  #     response.should == resp
-  #   end
-  # end
+  xit "fetches single videos" do
```
Comment (allenlsy) [inline spec/viki/video_spec.rb]: why are we skipping this spec?

File: lib/viki/version.rb (hunk 1 above)
Comment (piratefsh) [reply lib/viki/version.rb]: according to [semver](http://semver.org/#how-should-i-handle-deprecating-functionality), deprecation should warrant minor version bump. i am assuming we are following semver guidelines?

File: spec/viki/video_spec.rb (hunk 2 above)
Comment (piratefsh) [reply spec/viki/video_spec.rb]: not sure. it's been commented out all this while, i just changed commented out to xit

File: lib/viki/version.rb (hunk 1 above)
Comment (allenlsy) [reply lib/viki/version.rb]: 👍

File: spec/viki/video_spec.rb (hunk 2 above)
Comment (allenlsy) [reply spec/viki/video_spec.rb]: I see. cool

Comment (piratefsh) [reply spec/viki/video_spec.rb]: I ran the tests and they are passing. seems reasonable so i will add them back in `xit` => `it`.  Looks like they got commented during some refactor back in 2013.

**Comment by allenlsy** [comment]
➕ 1️⃣

**Pull request merged**